src/Plugin/migrate/process/AccessibleFile.php

```diff
@@ -99,6 +101,20 @@ public function transform($value, MigrateExecutableInterface $migrate_executable
     $allowed_paths = $this->configFactory->get('islandora_spreadsheet_ingest.settings')->get('binary_directory_whitelist');
     $allowed_schemes = $this->configFactory->get('islandora_spreadsheet_ingest.settings')->get('schemes');
 
+    // Ignore this check if relevant configuration has not been set, and advise
+    // that the configuration should be set.
+    if (empty($allowed_paths) && empty($allowed_schemes)) {
+      $migrate_executable->saveMessage(
+        strtr("Skipping this process to check (:file) since nothing has been configured to check against in the approved list of directories or schemes. For site security, it is highly recommended to set appropriate configuration at :config_url.", [
```

Comment:
Should probably avoid "this", as it only makes sense when looking at _this_ file, not so much when considering seeing the message on its own, say, when reviewing the messages table.

```suggestion
        strtr("File access criteria is not defined; passing check of file (:file). For site security, it is highly recommended to set appropriate configuration at :config_url.", [
```

Or something of the like?